.github/workflows/ci.yml

```diff
@@ -482,6 +482,10 @@ jobs:
             !:trino-sqlserver,
             !:trino-test-jdbc-compatibility-old-server,
             !:trino-tests'
+      - name: Maven FileSystemCache Tests
+        # Isolate HDFS file system cache concurrency test to avoid flakiness
+        run: |
+          $MAVEN test ${MAVEN_TEST} -pl :trino-hdfs -P test-hdfs-cache
```

Comment:
regardless of GIB?

what if test-hdfs-cache is removed from "other modules"?

Reply:
Not sufficiently versed with GIB, need to get back on this.